state/accountsDB.go

```diff
@@ -329,22 +372,31 @@ func (adb *AccountsDB) saveCode(newAcc, oldAcc baseAccountHandler) error {
 		newCodeHash = adb.hasher.Compute(string(newCode))
 	}
 
+	if adb.enableEpochsHandler.IsFlagEnabled(common.MigrateCodeLeafFlag) &&
```

Comment:
Extract these checks and L354->L362 in a new func, `isCodeMigration()` or something like that

Reply:
right, refactored this area